package.json

```diff
@@ -85,12 +85,12 @@
     },
     "./extensions/*": {
       "import": {
-        "types": "./build/esm/extensions/*",
-        "default": "./build/esm/extensions/*"
+        "types": "./build/esm/extensions/*/index.d.ts",
+        "default": "./build/esm/extensions/*/index.js"
       },
       "require": {
-        "types": "./build/cjs/extensions/*",
-        "default": "./build/cjs/extensions/*"
+        "types": "./build/cjs/extensions/*/index.d.ts",
```

Comment:
Hi! We’ve reviewed the pull request and have a couple of suggestions:  
1) We recommend using absolute paths to the `index` file in all extensions from `additional`.  
2) The same applies to the `view` folder.

Reply:
Example:

```
"./extensions/Mermaid": {
  "default": "./build/cjs/extensions/additional/Mermaid/index.js",
},
```

```
"./view/mermaid": {
  "default": "./build/cjs/view/mermaid.js", // here exports all things related to diplodoc/mermaid-extension in src/view
},
```